client/components/Login.js

```diff
@@ -1,20 +1,31 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 import { connect } from 'react-redux';
-import { getUser, getGithubUser } from '../store';
+import { getUser, getGithubUser, createUser } from '../store';
 
 const LoginClass = props => {
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
 
   const saveHandler = ev => {
     ev.preventDefault();
-    props.getUser({ email, password }).then(() => props.history.push('/'));
+    props
+      .getUser({ email, password })
+      .then(() => props.history.push('/'))
+      .catch(error => console.log(error));
   };
 
   const githubOauth = () => {
     // TODO: Figure out a better way to to do this, as it doesn't work on heroku.
-    // history.push?
-    window.location.href = 'http://localhost:3000/github/login';
+    // history.push? should fixed -kristy
+    window.location.href = `${window.location.origin}/github/login`;
+  };
+
+  const userSignUp = ev => {
+    ev.preventDefault();
+    props
+      .createUser({ email, password })
+      .then(() => props.history.push('/'))
```

Comment:
After logging in should redirect to challenges page.
```suggestion
      .then(() => props.history.push('/challenges'))
```